presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorMetadata.java

```diff
@@ -432,6 +425,8 @@ public ConnectorInsertTableHandle beginInsert(ConnectorSession session, Connecto
 
         long transactionId = shardManager.beginTransaction();
 
+        setTransactionId(transactionId);
```

Comment:
This is just converting the APIs with as few code changes as possible, following the same pattern as the adapter layer. It almost certainly will need to be part of the handle later.